### ISSUE TITLE
InputNumber does not accept typing boundary values

### ISSUE THREAD

**Issue by developerkam** [primefaces/primereact#1417]
### There is no guarantee in receiving an immediate response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeReact PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[X] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://forum.primefaces.org/viewforum.php?f=57
```

**Codesandbox Case (Bug Reports)**
Please fork the codesandbox below and create a case demonstrating your bug report. Issues without a codesandbox have much less possibility to be reviewed.

You can reproduce the problem even showcase/#/inputnumber page.

**Current behavior**
Boundary values cannot be typed, but the values can be set by buttons or up/down arrows.

**Expected behavior**
Boundary (min-max) values allowed to be typed.

**Minimal reproduction of the problem with instructions**
Simply put min and max properties to InputNumber and try to type.
<InputNumber value={this.state.value4} onChange={(e) => this.setState({value4: e.value})} mode="decimal" min={0} max={100} />

* **React version:**
16.12

* **PrimeReact version:**
4.2.2

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
All
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
All

**Comment by mertsincan** [comment]
Duplicate of https://github.com/primefaces/primereact/issues/1418

On next version, please use onValueChange={(e) => this.setState({componentValue: e.value})}

Best Regards,
